modules/noc/class_noc.php

```diff
@@ -24,6 +24,7 @@ class noc
 {
     public function MACtoIP($SearchMAC, $IP, $ReadCommunity)
     {
+        $IP = null;
```

Comment:
This appears to be not correct.
``$IP``  is passed as parameter and then set to null?

Reply:
Fixed in https://github.com/lansuite/lansuite/pull/583/commits/8284f31913a4974eb88b6b3641bafdf0fc8a1741